src/app/(blog-pages)/layout.tsx

```diff
@@ -0,0 +1,11 @@
+'use client'
+
+export default function Layout({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
+  return (
+    <div className="p-5 mx-auto w-[1300px]">{children}</div>
```

Comment:
```suggestion
    <div className="flex w-full min-h-screen">
      <Sidebar className="w-[300px] hidden lg:block border-r" />
      <div className="p-5 mx-auto w-[1300px]">{children}</div>;
    </div>
```

We remove the sidebar form root layout, so we need to add blog-pages layout.